README.md

```diff
@@ -78,6 +78,15 @@ Currently you must use a version of TensorFlow from
 instructions](#build-the-tensorflow-backend-with-custom-tensorflow)
 below.
 
+### How does the TensorFlow backend manage GPU memory?
+
+The TensorFlow backend does not "release" GPU memory until the Triton process
+exits. TensorFlow uses a pool allocator and so it retains any memory it
+allocates. It will reuse that memory if you load another TensorFlow model, but
+it will not return it to the system, even if it is no longer using it. For this
+reason, it is preferred to keep TensorFlow models grouped together on the same
+Triton process if you will be repeatedly loading/unloading them.
+
```

Comment:
Also offer a combination of [gpu-memory-fraction](https://github.com/triton-inference-server/tensorflow_backend#--backend-configtensorflowgpu-memory-fractionfloat) and [rate-limiter ](https://github.com/triton-inference-server/server/blob/main/docs/rate_limiter.md)as a work-around solution to keep the memory allocations in check.